src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -29,6 +29,39 @@ int init_sim() {
         return -1;
     }
 
+    // Configue tracing.
+    const char *T = getenv("HALIDE_HEX_SIM_MIN_TRACE");
+    if (T && T[0] != 0) {
+        status - sim->SetTracing(HEX_TRACE_PC_MIN, T);
+        if (status != HEX_STAT_SUCCESS) {
+            fprintf(stderr,"HexagonWrapper::SetTracing MIN failed: %d\n", statu
```

Comment:
Can this just use printf? Remember this isn't simulator side code running, so the buffering issues shouldn't be a problem. It's what the rest of the code in this file does.